seals/src/txout/blind.rs

```diff
@@ -337,142 +283,115 @@ mod test {
-                "tapret1st:646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#\
-                 0x78cs"
+            ChainBlindSeal::from_str(
+                "646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#0x78cs"
             ),
             Err(ParseError::WrongBlinding)
         );
         assert_eq!(
-            ChainBlindSeal::<CloseMethod>::from_str(
-                "tapret1st:646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#\
-                 78ca95"
+            ChainBlindSeal::from_str(
+                "646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#78ca95"
             ),
             Err(ParseError::NonHexBlinding)
         );
         assert_eq!(
-            ChainBlindSeal::<CloseMethod>::from_str(
-                "tapret1st:646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#857"
+            ChainBlindSeal::from_str(
+                "646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#857"
             ),
             Err(ParseError::NonHexBlinding)
         );
         assert_eq!(
-            ChainBlindSeal::<CloseMethod>::from_str(
-                "tapret1st:646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#-5"
+            ChainBlindSeal::from_str(
+                "646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#-5"
             ),
             Err(ParseError::NonHexBlinding)
         );
 
         // wrong txid value
         assert_eq!(
-            ChainBlindSeal::<CloseMethod>::from_str(
-                "tapret1st:646ca5c1062619e2a2d607719dfd820551fb773e4dc8c4ed67965a8d1fae839:5#\
-                 0x78ca69"
+            ChainBlindSeal::from_str(
+                "646ca5c1062619e2a2d607719dfd820551fb773e4dc8c4ed67965a8d1fae839:5#0x78ca69"
             ),
             Err(ParseError::WrongTxid(hex::Error::OddLengthString(63)))
         );
         assert_eq!(
-            ChainBlindSeal::<CloseMethod>::from_str("tapret1st:rvgbdg:5#0x78ca69"),
+            ChainBlindSeal::from_str("rvgbdg:5#0x78ca69"),
             Err(ParseError::WrongTxid(hex::Error::InvalidChar(b'r')))
         );
         assert_eq!(
-            ChainBlindSeal::<CloseMethod>::from_str(
-                "tapret1st:10@646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#\
-                 0x78ca69"
+            ChainBlindSeal::from_str(
+                "10@646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:5#0x78ca69"
             ),
             Err(ParseError::WrongTxid(hex::Error::OddLengthString(67)))
         );
 
         // wrong structure
         assert_eq!(
-            ChainBlindSeal::<CloseMethod>::from_str(
-                "tapret1st:646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:1"
+            ChainBlindSeal::from_str(
+                "646ca5c1062619e2a2d60771c9dfd820551fb773e4dc8c4ed67965a8d1fae839:1"
```

Comment:
The problem with this seal serialization format is that it is nohow different from `Outpoint` and easy to mix.

I think we need a prefix anyway, like `txoseal:` or something

Reply:
done